src/components/transverse/EnTete/EnTete.tsx

```diff
@@ -1,12 +1,14 @@
+'use client'
```

Comment:
Pourquoi cette ligne ?

Reply:
Par défaut, tout est server component sauf que ces derniers ne connaissent pas les hook donc il faut ajouter cette [signature](https://nextjs.org/docs/app/building-your-application/rendering/client-components#using-client-components-in-nextjs).